src/main/java/seedu/planner/ui/MainWindow.java

```diff
@@ -307,4 +313,10 @@ private void handleSuggestModule(SuggestModuleEvent event) {
                 event.getIndex(), ModulePanelType.SUGGESTED);
         suggestedModulesPlaceholder.getChildren().set(CURRENT_NODE, panel.getRoot());
     }
+
+    @Subscribe
+    private void handleListEvent(ListModuleEvent event) {
+        ModuleListPanel panel = takenModuleListPanels.get(MAX_NUMBER_SEMESTERS);
+        takenModulesPlaceholder.getChildren().set(CURRENT_NODE, panel.getRoot());
```

Comment:
The use of `CURRENT_NODE` is not really "safe" cos not panels have their child at index `1`. My mistake. Best to use `MainWindow#setPlaceHolder` in PR I merged